examples/src/ref_finance.rs

```diff
@@ -71,7 +71,7 @@ async fn create_wnear(owner: &Account, worker: &Worker<Sandbox>) -> anyhow::Resu
         .call(wnear.id(), "new")
         .args_json(json!({
             "owner_id": owner.id(),
-            "total_supply": parse_near!("1,000,000,000 N"),
+            "total_supply": parse_near!("1,000,000,000 N").to_string(),
         }))
```

Comment:
`wrap.near` contract `new` method does not expect any arguments O_o

https://github.com/near/core-contracts/blob/3f3170fce91ff4d8c6ee9d15683f2d4dfe1275cf/w-near/src/lib.rs#L36

Let's remove `args_json` completely.

Reply:
OK, let me try!

Reply:
It works okay!